### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by slavidex**

### CODE REVIEW

File: app/src/main/res/values-bg/strings.xml [hunk 1]
```diff
@@ -375,7 +375,7 @@
     <string name="resend_all_data">Изпрати всички данни отново</string>
     <string name="wear">Wear</string>
     <string name="wrongpumpdriverselected">Избран е грешен тип помпа</string>
-     <string name="minago">преди %d м.</string>
+     <string name="minago" formatted="false">преди %d м.</string>
     <string name="actions_shortname">" "</string>
```
Comment (MilosKozak) [inline app/src/main/res/values-bg/strings.xml]: You must remove these translated

**Comment by slavidex** [comment]
perhaps as it was before?

**Comment by MilosKozak** [comment]
sorry that was correct. lines like actions_shortname are twice there

**Comment by slavidex** [comment]
I use a copy and paste. Sorry about that

**Pull request merged**